lib/structures/Member.js

```diff
@@ -57,7 +57,7 @@ class Member extends Base {
         } else {
             this.user = null;
         }
-        this.status = "offline";
+
```

Comment:
This might need a typings change 🤔 For status: to status?:

Reply:
Addressed that, I had to make status optional in `Status` interface as well. Not sure if that's the correct way to do, I'll let you have a look

Reply:
I am busy this weekend so unable to give a thorough look through eris/api docs, take this with a grain of salt. 

I would have just removed the `implements Presence` on the Member tbh. I am not sure Discord would ever send a Presence which was undefined status so idk how accurate that is to mark it also as optional. I would wait for @alex-taxiera to chime in here as he has the best experience with Presence related stuff and it would affect his project most of all.

Reply:
As the sole protector of presence data, I find no issue with this. All of the other presence related data is already marked as nullable anyhow, and really this interface was built to be implemented, not used as it's own data structure.